docs/src/main/tut/user-guide.md

```diff
@@ -803,6 +803,116 @@ implicit val encodeExceptionCirce: Encoder[Exception] = Encoder.instance(e =>
 If no other `Encode[Exception]` is available, Finch provides a fallthrough of `Encode.Aux[Exception, ?]`
 that will return an empty content body.
 
+### Streaming
+
+`finch-iteratee` module enables high-level support of chunked request and response streaming using
+[iteratee.io](https://github.com/travisbrown/iteratee) and `circe-streaming` libraries. It allows to encode and decode
```

Comment:
Maybe `It allows encoding and decoding newline delimited JSON streams, but also supports binary and text streaming.`

Also s/delimeted/delimited whether we like my above suggestion or not.